bip-musig2.mediawiki

```diff
@@ -207,6 +207,11 @@ This specification is written with a focus on clarity.
 As a result, the specified algorithms are not always optimal in terms of computation and space.
 In particular, some values are recomputed but can be cached in actual implementations (see [[#signing-flow|Signing Flow]]).
 
+The specification also attempts to prevent vulnerabilities that result from accidental mixups of different sessions.
+For example, using a ''pk'' argument in the ''NonceGen'' algorithm that does not match the ''sk'' later provided in ''Sign'' is insecure (see [https://github.com/jonasnick/musig2-tweaking here] for the details).
```

Comment:
We can't really use this in Sign because Sign needs local variable `d' = int(sk)`. We could in principle let PlainPublicKey accept an integer input but that seems suboptimal.

Reply:
Let's do this after this PR maybe.